op-chain-ops/upgrades/l1.go

```diff
@@ -384,13 +382,38 @@ func OptimismMintableERC20Factory(batch *safe.Batch, implementations superchain.
 		}
 	}
 
+	OptimismMintableERC20FactoryABI, err := bindings.OptimismMintableERC20FactoryMetaData.GetAbi()
+	if err != nil {
+		return err
+	}
+
+	var bridge common.Address
+	if config != nil {
```

Comment:
I am slightly confused about this `config != nil` check as the branch doesn't utilize the config

Reply:
`config` is passed as parameter to the function (could be `nil`) and implemented this pattern for consistency with https://github.com/ethereum-optimism/optimism/blob/e68a5baa2ec7f62b1ea30dfacca23b345fc795f1/op-chain-ops/upgrades/l1.go#L307 and https://github.com/ethereum-optimism/optimism/blob/e68a5baa2ec7f62b1ea30dfacca23b345fc795f1/op-chain-ops/upgrades/l1.go#L586. what are your thoughts @tynes? can remove if you think so

Reply:
Across all of the PRs, we should adopt a pattern of always using the remote values and asserting that the values match what is in the config if the config is present. We should log if there is no config so we know that the checking was skipped